pkgs/applications/networking/irc/iim/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, stdenv }:
+stdenv.mkDerivation {
```

```suggestion
{ lib, stdenv, fetchFromGitHub }:

stdenv.mkDerivation {
```

.. would be added here.